enhancements/windows-containers/disconnected-support.md

```diff
@@ -0,0 +1,355 @@
+To support disconnected environments, all images need to be pulled from air-gapped mirror registries, whether that be 
+the OpenShift internal image registry or other ones on-prem.
+
+There already exists a protocol for users to publish [registry mirroring configuration](https://docs.openshift.com/container-platform/4.14/openshift_images/image-configuration.html#images-configuration-registry-mirror_image-configuration),
+namely `ImageDigestMirrorSet` (IDMS), `ImageTagMirrorSet` (ITMS) cluster resources. 
+These are consumed by OpenShift components like the Machine Config Operator (MCO) to apply the settings to Linux infra
+and worker nodes, Windows worker nodes do not currently consume or respect mirror registry settings when pulling images.
+This effort will work to plug feature disparity by making the [Windows Machine Config Operator](https://github.com/openshift/windows-machine-config-operator)
+(WMCO) aware of mirror registry settings at operator install time and reactive during its runtime.
+
+
+## Motivation
+
+The motivation here is to expand the Windows containers production use case, enabling users to add Windows nodes and run
+workloads easily and successfully in a disconnected clusters. This is an extremely important ask for customer
+environments where Windows nodes need to pull images from air-gapped, on-prem registries for security reasons.
+The added benefit here is that the changes proposed in this enhancement are not restricted to disconnected environments;
+users in __connected__ clusters can take advantage of this mirror registry support.
+
+### Goals
+
+* Create an automated mechanism for WMCO to consume global mirror registry settings from existing platform resources,
+  such that it enables Windows nodes to pull images in disconnected environments
+* Configure the mirror registry settings in WMCO-managed components on Windows nodes, namely the containerd runtime
+* React to changes to the mirror registry settings during WMCO runtime
+* Maintain normal functionality in connected clusters and clusters without any cluster-wide mirror registry configuration
+
+### Non-Goals
+
+* First-class support of [all OpenShift image registry setting configurations](https://docs.openshift.com/container-platform/4.14/openshift_images/image-configuration.html#images-configuration-file_image-configuration), including the `Image` resource
+* Supporting clusters that use only `ImageContentSourcePolicy` resources. These were [deprecated in 4.13](https://docs.openshift.com/container-platform/4.14/release_notes/ocp-4-14-release-notes.html#ocp-4-14-deprecated-removed-features) 
+and users [should migrate to IDMSes](https://docs.openshift.com/container-platform/4.14/openshift_images/image-configuration.html#images-configuration-registry-mirror-convert_image-configuration) instead.
+* Using [`ImageContentPolicy` resources](https://docs.openshift.com/container-platform/4.14/rest_api/config_apis/imagecontentpolicy-config-openshift-io-v1.html)
```

Comment:
Out of scope for the disconnected MVP

Reply:
I understand this is out of scope but it will help to state why. This is going to be an obvious follow up question from a customer and product management and it will be good to have it documented.